### PR TITLE
[fix](nereids) rf not generated after postprocessor

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes
RecomputeLogicalPropertiesProcessor changes plan node, but does not update rf target and src. It makes rf become invalid, and not translated properly.

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

File: regression-test/pipeline/p0/conf/regression-conf.groovy [hunk 1]
```diff
@@ -55,10 +55,10 @@ testDirectories = ""
 excludeGroups = ""
 // this suites will not be executed
 
-excludeSuites = "test_bitmap_index,test_ddl,test_analyze,test_leading,test_stream_load_move_memtable,ds_rf95,test_profile,test_broker_load,test_spark_load,test_refresh_mtmv,test_bitmap_filter,test_export_parquet,test_doris_jdbc_catalog,nereids_delete_mow_partial_update"
+excludeSuites = "test_bitmap_index,test_ddl,test_analyze,test_leading,test_stream_load_move_memtable,ds_rf95ds_rf16,ds_rf94,test_profile,test_broker_load,test_spark_load,test_refresh_mtmv,test_bitmap_filter,test_export_parquet,test_doris_jdbc_catalog,nereids_delete_mow_partial_update"
```
Comment (xzj7019) [inline regression-test/pipeline/p0/conf/regression-conf.groovy:58]: 1）typo? ds_rf95ds_rf16
2) why exclude 95/94/16 ?

Comment (englefly) [reply regression-test/pipeline/p0/conf/regression-conf.groovy:58]: 这几个在变，先把其他的query 盯住

**Comment by englefly** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**